extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -321,6 +321,12 @@ export class AmpA4A extends AMP.BaseElement {
      * @private {boolean}
      */
     this.isCollapsed_ = false;
+
+    /**
+     * The outer most iframe loaded by this ad slot.
```

Comment:
Outer most?  Not sure I know what you mean.  How about instead:  "Frame in which the creative renders (friendly if validated AMP, xdomain otherwise)"